controllers/watcher_controller.go

```diff
@@ -69,35 +71,45 @@ func (r *WatcherReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	watcherObj := &v1alpha1.Watcher{}
 	if err := r.Get(ctx, client.ObjectKey{Name: req.Name, Namespace: req.Namespace}, watcherObj); err != nil {
 		logger.Error(err, "Failed to get reconciliation object")
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+		return ctrl.Result{Requeue: true}, client.IgnoreNotFound(err)
```

Comment:
if you have a no found it will do error == nil but still requeue. I think this will infinetly requeue then.

Reply:
sure, this requeue is not need 👍